spec/requests/todos_spec.rb

```diff
@@ -41,29 +94,109 @@
   # 異常系
   describe '異常系' do
     describe 'GET #show' do
-      context '存在しないTODOにアクセスしたとき' do
+      let!(:todo) { create(:todo) }
+
+      before do
+        allow(Todo).to receive(:find).and_raise(error)
+        get "/api/todos/#{todo.id}"
+      end
+
+      context 'レコードが存在しない場合' do
+        let!(:error) { ActiveRecord::RecordNotFound.new }
+        it 'エラーになること' do
+          expect(response).to have_http_status(404)
+        end
+      end
+
+      context 'レコードが存在しない場合' do
+        let!(:error) { Exception.new }
+        it 'エラーになること' do
+          expect(response).to have_http_status(500)
+        end
+      end
+    end
+
+    describe 'GET #index' do
+      let!(:todo) { create(:todo) }
+
+      before do
+        allow(Todo).to receive(:all).and_raise(error)
+        get "/api/todos"
+      end
+
+      context 'レコードが存在しない場合' do
+        let!(:error) { ActiveRecord::RecordNotFound.new }
+        it 'エラーになること' do
```

Comment:
```suggestion
        it 'statusが404になること' do
```

上記のような感じのテストケース名の方がいいかもです
同様箇所あり

Reply:
ありがとうございます！
500エラーも含めて下記のように修正してみました。

[requests_todos_異常系_テストケース名を具体的なステータス名に修正](https://github.com/aki366/TODO_api/pull/14/commits/0783a1a00e8df3e9148be3a33f89de468aa39fdb) https://github.com/aki366/TODO_api/issues/5

```
  異常系
    GET #show
      レコードが存在しない場合
        statusが404になること
      Excepionエラーの場合
        statusが500になること
    GET #index
      レコードが存在しない場合
        statusが404になること
      Excepionエラーの場合
        statusが500になること
    PUT #update
      レコードが存在しない場合
        statusが404になること
      Excepionエラーの場合
        statusが500になること
    POST #create
      Excepionエラーの場合
        statusが500になること
    DELETE #destroy
      レコードが存在しない場合
        statusが404になること
      Excepionエラーの場合
        statusが500になること
```